src/components/layout/MainNavigation.tsx

```diff
@@ -0,0 +1,24 @@
+import Logo from './Logo.jsx'
+import { NavLink, Link } from 'react-router-dom'
+
+const MainNavigation = () => {
+    const activeClass = 'bg-primary text-green font-nav px-3 py-1 text-[30px] '
+    const linkClass = 'font-nav px-3 py-1 text-[30px] hover:bg-primary hover:text-green'
+    const navLinks = [
```

Comment:
ooh, matrices? This is not a usual pattern, make these object in the array, like 

type Link = {
 title: string
 url: string
}
and then NavLinks : Link[] = [
{ title: "Home", url: "/"
}
] etc

Reply:
ok done!

Reply:
I always tend to forget to define type haha